packages/rolldown/src/cli/env.ts

```diff
@@ -0,0 +1,28 @@
+import tty from 'node:tty'
+
+/**
+ * The following flags are based on conosola
+ * https://github.com/unjs/consola/blob/24c98ceb90c269a170fd116134d91803a89f2c9d/src/utils/color.ts#L7-L26
```

Comment:
@pi0
I would like to use `std-env` for these flags, but it seemed that tree-shaking was not very optimized for it. If you use `std-env` and can bundle only the necessary flags code, I would like to know.

If `std-env` is not optimized for tree-shaking, I can contribute to it.

Reply:
std-env has ESM exports, is ~3.58kB in total and I tried to make the dependency graph as less coupled as possible.

Still, if you believe more tree-shaking is possible, please open an upstream issue explaining what exact import names you believe are not necessary bits and are not tree-shaken. 🙏🏼

Reply:
Thank you for your comment! 😄 
I'll try to open as issue and share the my ideas to resolve it.